pulp/apis/cuopt_api.py

```diff
@@ -0,0 +1,337 @@
+import ctypes
+import os
+import subprocess
+import sys
+import warnings
+from uuid import uuid4
+import numpy as np
```

Comment:
numpy is not part of PuLP dependencies. So this import will fail if cuopt is not installed.

Reply:
Moved to import while importing cuopt in #839